### ISSUE TITLE
training iterations

### ISSUE THREAD

**Issue by sujyQ** [TencentARC/MM-RealSR#14]
Hi, @MC-E. I really thank for your awesome work.

I have some questions about training MM-RealSR.

1. In your paper, MM-RealSR was trained during 1000k iterations for phase 1 and 400k iters for phase 2, which are different with .yml files. How can i reproduce the results?

2. How long did it take with your env?

Thanks!

**Comment by MC-E** [comment]
Hi, Phase 1 trains the network with MSE loss, and Phase 2 is trained with the GAN loss with the initial parameters from Phase 1. About 7 days to complete these two phases.

**Comment by sujyQ** [comment]
Thanks for reply, but I wondered the number of iterations for each phase, not loss function:)
The "total_iter" in .yml files are not same as the paper.

**Comment by MC-E** [comment]
Sorry for the misunderstanding. We did some debugging before releasing the code. We found that it is unnecessary to train the net stage for too long, so we shortened the number of iterations for it and appropriately increased the number of training steps for the GAN stage. This has little impact on the quantitative metrics.
